As the korina ethernet driver uses platform_get_drvdata() to extract the
driver specific data from the platform device, driver_data has to be
used here.

Signed-off-by: Phil Sutter <n0-1@freewrt.org>
---
 arch/mips/rb532/devices.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

--- a/arch/mips/rb532/devices.c
+++ b/arch/mips/rb532/devices.c
@@ -86,7 +86,7 @@ static struct korina_device korina_dev0_
 static struct platform_device korina_dev0 = {
 	.id = -1,
 	.name = "korina",
-	.dev.platform_data = &korina_dev0_data,
+	.dev.driver_data = &korina_dev0_data,
 	.resource = korina_dev0_res,
 	.num_resources = ARRAY_SIZE(korina_dev0_res),
 };
